src/PhpDocument.php

```diff
@@ -70,9 +89,36 @@ public function __construct(string $uri, Project $project, LanguageClient $clien
      *
      * @return SymbolInformation[]
      */
-    public function getSymbols()
+    public function getSymbols(): array
     {
-        return $this->symbols;
+        $nodeSymbolKindMap = [
+            Node\Stmt\Class_::class           => SymbolKind::CLASS_,
+            Node\Stmt\Trait_::class           => SymbolKind::CLASS_,
+            Node\Stmt\Interface_::class       => SymbolKind::INTERFACE,
+            Node\Stmt\Namespace_::class       => SymbolKind::NAMESPACE,
+            Node\Stmt\Function_::class        => SymbolKind::FUNCTION,
+            Node\Stmt\ClassMethod::class      => SymbolKind::METHOD,
+            Node\Stmt\PropertyProperty::class => SymbolKind::PROPERTY,
+            Node\Const_::class                => SymbolKind::CONSTANT
+        ];
+        $symbols = [];
+        foreach ($this->definitions as $fqn => $node) {
```

Comment:
I can reproduce too. But it is weird because `parse()` sets it to an array and `getSymbols()` never runs before `parse()`.